tests/bluetooth/audio/cap_commander/src/test_volume.c

```diff
@@ -0,0 +1,552 @@
+/* test_change_volume.c - unit test for volume settings */
+
+/*
+ * Copyright (c) 2023-2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/fff.h>
+
+#include "bluetooth.h"
+#include "cap_commander.h"
+#include "conn.h"
+#include "expects_util.h"
+#include "cap_mocks.h"
+
+#define FFF_GLOBALS
+
+struct cap_commander_test_vol_fixture {
+	struct bt_conn conns[CONFIG_BT_MAX_CONN];
+};
+
+static void test_conn_init(struct bt_conn *conn)
+{
+	conn->index = 0;
+	conn->info.type = BT_CONN_TYPE_LE;
+	conn->info.role = BT_CONN_ROLE_PERIPHERAL;
+	conn->info.state = BT_CONN_STATE_CONNECTED;
+	conn->info.security.level = BT_SECURITY_L2;
+	conn->info.security.enc_key_size = BT_ENC_KEY_SIZE_MAX;
+	conn->info.security.flags = BT_SECURITY_FLAG_OOB | BT_SECURITY_FLAG_SC;
+}
+
+static void cap_commander_test_vol_fixture_init(struct cap_commander_test_vol_fixture *fixture)
+{
+	for (size_t i = 0; i < ARRAY_SIZE(fixture->conns); i++) {
+		test_conn_init(&fixture->conns[i]);
+	}
+}
```

Comment:
I think it is OK that each file has its own fixture, but suggest to move the `test_conn_init` to a common file for CAP (as that won't differ between each feature).